.github/workflows/main.yml

```diff
@@ -239,6 +239,22 @@ jobs:
   #       WBINDGEN_I_PROMISE_JS_SYNTAX_WORKS_IN_NODE: 1
   #   - run: cargo build --manifest-path crates/web-sys/Cargo.toml --target wasm32-unknown-unknown --features "Node Window Document"
 
+  # This checks that the output of the CLI is actually valid JavaScript and TypeScript
+  test_cli_reference_typescript:
+    name: Run CLI reference TypeScript check
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+    - uses: actions/setup-node@v4
+      with:
+        node-version: '20'
+    - run: npm i -g typescript
+    - run: npm i --save @types/node @types/deno
+    - name: Check TypeScript output
+      run: tsc --noEmit --skipLibCheck --lib esnext,dom $(echo crates/cli/tests/reference/*.d.ts)
```

Comment:
Whats going on with the `echo` call here?

Reply:
`tsc` doesn't support glob patterns. To pass multiple files to `tsc`, you either have to add a `tsconfig.json` (which isn't possible, because that would make it impossible to check `.js` and `.d.ts` separately) or pass in all files separated by spaces. The `echo` call does exactly that. It runs the glob pattern and returns the paths of all matching files separated by spaces.